### PR TITLE
fix: missing types

### PR DESCRIPTION
**Pull request by eddieantonio**
There were missing types for `TransducerFile.lookup_symbols()`, so I added them.

I also:

 * upgraded to mypy 0.800 — no need for `from __future__ import annotations` 🎉 
 * added missing type signatures in `import_test.py`; it's really annoying, but mypy will NOT check types unless there is at least one annotated parameter or if the return type is annotated. So I added a bunch of `-> None` to fix this :/

### CODE REVIEW

File: Pipfile [hunk 1]
```diff
@@ -4,6 +4,7 @@ verify_ssl = true
 name = "pypi"
 
 [packages]
+mypy = ">=0.800"
```
Comment (andrewdotn) [inline Pipfile:7]: Can we update the `mypy` under `dev-packages` instead?

**Review by andrewdotn — COMMENTED** [review]
Oops, yup. Would you be ok with this `mypy.ini` to help catch this in the future?

```
[mypy]
follow_imports = skip
show_error_codes = True
allow_untyped_decorators = True # for pytest
pretty = True
strict = True
```

**Comment by eddieantonio** [comment]
> Oops, yup. Would you be ok with this `mypy.ini` to help catch this in the future?
> 
> ```
> [mypy]
> follow_imports = skip
> show_error_codes = True
> allow_untyped_decorators = True # for pytest
> pretty = True
> strict = True
> ```

Did you mean [`disallow_untyped_defs
`](https://mypy.readthedocs.io/en/stable/config_file.html#confval-disallow_untyped_defs)

**Comment by eddieantonio** [comment]
> > Oops, yup. Would you be ok with this `mypy.ini` to help catch this in the future?
> > ```
> > [mypy]
> > follow_imports = skip
> > show_error_codes = True
> > allow_untyped_decorators = True # for pytest
> > pretty = True
> > strict = True
> > ```
> 
> Did you mean [`disallow_untyped_defs `](https://mypy.readthedocs.io/en/stable/config_file.html#confval-disallow_untyped_defs)

@andrewdotn Never mind, I figured out which options to use!

**Review by andrewdotn — APPROVED** [review]

**Pull request merged**